wrapper.py

```diff
@@ -331,3 +330,16 @@ def load_h5(filename, path='', lazy=False):
                   "will be removed in the next release. Please use load() instead.",
                   DeprecationWarning)
     return load(filename, path=path, lazy=lazy)
+
+
+# Helper function
+def _accumulate(iterator):
+    """
+    Create a generator to iterate over the accumulated
+    values of the given iterator.
+    """
+    total = 0
+    yield total
+    for item in iterator:
+        total += item
```

Comment:
if you switch `total += item` and `yield total` you don't need the `yield` before the loop.